docs/core/tools/dotnet-add-reference.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: dotnet-add reference command
+title: dotnet add reference command
 description: The dotnet add reference command provides a convenient option to add project to project references.
 ms.date: 06/26/2019
 ---
-# dotnet-add reference
+# dotnet add reference
```

Comment:
To avoid localization.

```suggestion
# `dotnet add` reference
```

Reply:
I'll test this on a separate branch since I don't use this in any of the commands. We should do for all if we're going to do this.

Reply:
@mairaw, Thanks for pointing out that all the commands doesn't have these backticks. I saw those articles and most of them doesn't get localized. But found this one that got localized.

https://docs.microsoft.com/tr-tr/dotnet/core/tools/dotnet-store

![image](https://user-images.githubusercontent.com/31348972/67833780-4e0ac580-faee-11e9-8ea7-67f077cdd5ac.png)